src/health_misinfo_shared/fine_tuning.py

```diff
@@ -356,14 +444,16 @@ def save_all_responses(
         for claim in response.get("response", []):
             claims.append(claim.get("claim"))
             chunks.append(response.get("chunk"))
+            print("CLAIM", claim.get("claim"))
```

Comment:
I know they're handy for debugging, but can we remove all the `print` statements before deployment?